juno/assetlist.json

```diff
@@ -1879,6 +1879,33 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/bits.png"
         }
       ]
+    },
+    {
+      "description": "The least memable memecoin in existence. The cats have had their day, it's time for $POIL coin to take reign.",
```

Comment:
do you mean 'memeable' instead of 'memable'?

Reply:
Yes, good catch.
Updated the description in the assetlist and cw20 contract.